nvflare/ha/ha_admin_cmds.py

```diff
@@ -70,60 +70,73 @@ def list_sp(self, args, ctx: CommandContext):
         """
         overseer_agent = self.overseer_agent
         return {
-            "status": APIStatus.SUCCESS,
-            "details": str(overseer_agent.overseer_info),
-            "data": overseer_agent.overseer_info,
+            ProtoKey.STATUS: APIStatus.SUCCESS,
+            ProtoKey.DETAILS: str(overseer_agent.overseer_info),
+            ProtoKey.DATA: overseer_agent.overseer_info,
         }
 
     def get_active_sp(self, args, ctx: CommandContext):
         overseer_agent = self.overseer_agent
-        return {"status": APIStatus.SUCCESS, "details": str(overseer_agent.get_primary_sp())}
+        sp = overseer_agent.get_primary_sp()
+        return {ProtoKey.STATUS: APIStatus.SUCCESS, ProtoKey.DETAILS: str(sp), ProtoKey.META: sp.__dict__}
 
     def promote_sp(self, args, ctx: CommandContext):
         overseer_agent = self.overseer_agent
         if len(args) != 2:
-            return {"status": APIStatus.ERROR_SYNTAX, "details": "usage: promote_sp example1.com:8002:8003"}
+            return {
+                ProtoKey.STATUS: APIStatus.ERROR_SYNTAX,
+                ProtoKey.DETAILS: "usage: promote_sp example1.com:8002:8003",
+                ProtoKey.META: {MetaKey.STATUS: MetaStatusValue.SYNTAX_ERROR},
```

Comment:
should we also use "make_meta" method here?

Reply:
Yes

Reply:
done as suggested